### ISSUE TITLE
Tabs function for Electron apps.

### ISSUE THREAD

**Issue by sqwwqw5** [webrecorder/replayweb.page#148]
I know this feels more and more like making the replayweb.page into a standalone browser but considering the contents are websites, I think it is necessary to have the basic browser functions ready for users to enjoy browsing their beloved archived content.

**Comment by ikreymer** [comment]
Yes, this is definitely in direction of implementing a full-fledged browser, which we haven't had time to do.
Are you able to use the web-based version of ReplayWeb.page, or possibly, install it as a PWA? As a PWA, its basically a standalone app, but also comes with support for multiple windows (and search as well).
